crates/fda/src/main.rs

```diff
@@ -863,6 +868,7 @@ async fn pipeline(action: PipelineAction, client: Client) {
                     1,
                 ))
                 .map(|response| response.runtime_config.clone())
+                .unwrap()
```

Comment:
first unwrap() was already there, which is the way send() errors are bubbled up in fda (?). Second unwrap() added is because `runtime_config` is now an optional field.